go/go.sum

```diff
@@ -0,0 +1,39 @@
+github.com/BurntSushi/toml v1.2.1 h1:9F2/+DoOYIOksmaJFPw1tGFy1eDnIJXg+UHjuD8lTak=
```

Comment:
why freeze package versions?

Reply:
It looks like there isn't a way to tell go.mod to always pick up the latest version of a dependency, see [here](https://stackoverflow.com/a/68273838).

Reply:
I see there's an issue to add dependabot alerts [here](https://github.com/aws/glide-for-redis/issues/930), perhaps the best thing to do would be to set up dependabot alerts for our go dependencies and update when the alerts pop up

Reply:
or you could've just .gitignored the go.sum file 🤷